rosbag2_transport/test/rosbag2_transport/test_record_all_use_sim_time.cpp

```diff
@@ -52,7 +40,9 @@ TEST_F(RecordIntegrationTestFixture, record_all_with_sim_time)
 
   // clock
   std::string clock_topic = "/clock";
-  auto clock_message = get_clock_messages()[0];
+  rosgraph_msgs::msg::Clock::SharedPtr clock_message;
```

Comment:
By default shared pointer is used to be assigned to null pointer if no constructor specified.
Most probably it will be access by null address in next line and syshalt.  
It will be safer to specify constructor explicitly.
```cpp
auto clock_message = std::make_shared<rosgraph_msgs::msg::Clock>();
```

Reply:
Thank you. I've fixed the code.
c77926d